f5-bigip-mixin/dashboards/f5-cluster-overview.libsonnet

```diff
@@ -0,0 +1,633 @@
+local grafana = (import 'grafonnet/grafana.libsonnet');
+local dashboard = grafana.dashboard;
+local template = grafana.template;
+local prometheus = grafana.prometheus;
+local dashboardUid = 'f5-cluster-overview';
+
+local promDatasourceName = 'datasource';
+
+local promDatasource = {
+  uid: '${%s}' % promDatasourceName,
+};
+
+local nodeAvailabilityPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      '100 * sum by(partition, instance) (bigip_node_status_availability_state{job=~"$job", instance=~"$instance"})  / clamp_min(count by(partition, instance) (bigip_node_status_availability_state{job=~"$job", instance=~"$instance"}),1)',
+      datasource=promDatasource,
+      legendFormat='{{partition}} - {{instance}}',
+      format='time_series',
+    ),
+  ],
+  type: 'stat',
+  title: 'Node availability',
+  description: 'The percentage of nodes available.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'red',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 95,
+          },
+          {
+            color: 'yellow',
+            value: 96,
+          },
+          {
+            color: 'green',
+            value: 100,
+          },
+        ],
+      },
+      unit: 'percent',
+    },
+    overrides: [],
+  },
+  options: {
+    colorMode: 'value',
+    graphMode: 'none',
+    justifyMode: 'auto',
+    orientation: 'auto',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    textMode: 'auto',
+  },
+  pluginVersion: '10.2.0-60139',
+};
+
+local poolAvailabilityPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      '100 * sum by(partition, instance) (bigip_pool_status_availability_state{job=~"$job", instance=~"$instance"}) / clamp_min(count by(partition, instance)  (bigip_pool_status_availability_state{job=~"$job", instance=~"$instance"}),1)',
```

Comment:
just double checking that this state metric can only be 0 or 1 (i recall there being more than 2 states)

Reply:
The [bigip exporter](https://github.com/ExpressenAB/bigip_exporter/blob/e0f009c578b6264ea6c2fb82071ceb02a1a8121b/collector/vs_collector.go#L450) looks for "available" and returns 1 if found. Everything else is a 0 in this case.